project-orga/MAINTAINERS.md

```diff
@@ -0,0 +1,148 @@
+# The openHAB Maintainer manual
```

Comment:
There is a community discussion about the outdated aspects of these files in:

https://community.openhab.org/t/maintainers-files-outdated-or-not-present-in-nearly-all-github-projects/43266